src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+  display: block;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars.stars--5 .stars__star {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Need to fix
<img width="743" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/22ff5c9d-04f7-4ac3-8c1c-9b726a1da3d9">